### PR TITLE
[Caffe Frontend] Add support for Permute layer

### PR DESCRIPTION
**Pull request by mshr-h**
Added Permute layer support.
Related to #6206, #9060

@fernchen @zhiics @tqchen @FrozenGene
Can you take a look at this PR, please? Thanks!

### CODE REVIEW

File: tests/python/frontend/caffe/test_forward.py [hunk 1]
```diff
@@ -586,6 +586,27 @@ def test_forward_LRN():
     _test_lrn(data, local_size=3, alpha=2.0, beta=0.5, k=2.0)
 
 
+#######################################################################
+# Permute
+# -----------
```
Comment (FrozenGene) [inline tests/python/frontend/caffe/test_forward.py:591]: Alignment

**Review by FrozenGene — APPROVED** [review]

File: tests/python/frontend/caffe/test_forward.py (hunk 1 above)
Comment (mshr-h) [reply tests/python/frontend/caffe/test_forward.py:591]: @FrozenGene 
Sorry, can you specifically point out what's wrong with the code?
There seems to be no problem.

**Review by fernchen — APPROVED** [review]

**Comment by mshr-h** [comment]
Since #9060 was merged, `tvm-ci/pr-merge` should be passing correctly.

**Comment by masahi** [comment]
@mshr-h please take a look at the CI problem

**Comment by mshr-h** [comment]
hi @masahi thank you for your reply.
I re-run CI and everything has passed.

**Pull request merged**